src/printer/cvc/cvc_printer.cpp

```diff
@@ -179,7 +180,9 @@ void CvcPrinter::toStream(
       break;
 
     case kind::DATATYPE_TYPE: {
-      const Datatype& dt = (NodeManager::currentNM()->getDatatypeForIndex( n.getConst< DatatypeIndexConstant >().getIndex() ));
+      const Datatype& dt =
```

Comment:
Couldn't we change this code to use the `Node` layer?

Reply:
No, we have to consider whether the datatype was declared to be a `Record`, which is a user-level only field.

Reply:
Ah right, `isRecord()` has been removed from the Node layer.